cves/CVE-2018-8014.yml

```diff
@@ -145,8 +166,14 @@ interesting_commits:
     section by explaining what happened between the VCCs and the fix.
   answer:
   commits:
-    - commit:
-      note:
+    - commit: 
+      note: |
+        There were no interesting commits within the fixes of this vulnerability.
```

Comment:
Could you justify why there are no interesting commits? Maybe all the commits are doing something unrelated for example?

Reply:
The only commits related to this CVE are the direct fixes for each version of Tomcat.